app/printer.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+
+from app.book import Book
+
+
+class Printer(ABC):
+    def __init__(self, book: Book) -> None:
+        self.book = book
+
+    @abstractmethod
+    def print(self) -> None:
+        pass
+
+
+class ConsolePrinter(Printer):
+    def print(self) -> None:

```

Comment:
It is bad idea to use the names of built-in Python objects for your custom objects, correct everywhere.